src/examples/mfem_common_plane.cpp

```diff
@@ -0,0 +1,429 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Tribol Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (MIT)
+
+#include <set>
+
+#ifdef TRIBOL_USE_UMPIRE
+// Umpire includes
+#include "umpire/ResourceManager.hpp"
+#endif
+
+// MFEM includes
+#include "mfem.hpp"
+
+// Axom includes
+#include "axom/CLI11.hpp"
+#include "axom/slic.hpp"
+
+// Redecomp includes
+#include "redecomp/redecomp.hpp"
+
+// Tribol includes
+#include "tribol/common/Parameters.hpp"
+#include "tribol/config.hpp"
+#include "tribol/interface/tribol.hpp"
+
+/// Simple central difference method.
+class CentralDiffSolver : public mfem::SecondOrderODESolver
+{
+public:
+  CentralDiffSolver(const mfem::Array<int>& bc_vdofs_)
+  : bc_vdofs { bc_vdofs_ },
+    first_step { true } {}
+
+  void Step(mfem::Vector& x, mfem::Vector& dxdt, double& t, double& dt) override
```

Comment:
Self explanatory, but when pulling these classes out into new file, consider adding more comments on purpose of function and arguments.

Reply:
Done